packages/sites/src/pages/update-page.ts

```diff
@@ -23,9 +23,18 @@ import { removeUnusedResources } from "../layout/remove-unused-resources";
  */
 export function updatePage(
   model: IModel,
-  patchList: string[],
-  requestOptions: IHubRequestOptions
+  maybePatchList: string[] | Record<string, any>,
+  maybeRequestOptions?: IHubRequestOptions
 ): Promise<IUpdateItemResponse> {
+  let patchList = maybePatchList as string[];
```

Comment:
add a comment above this: `// by default assume the deprecated signature was used`